src/dataAssimilation/FieldIO.cpp

```diff
@@ -0,0 +1,312 @@
+/// \file       FieldIO.cpp
+/// \brief      Class for reading/writing the raw data of the fields u, v, w, p, T, C
+/// \date       Apr 18, 2021
+/// \author     My Linh Wuerzburger
+/// \copyright  <2015-2021> Forschungszentrum Juelich All rights reserved.
+
+#include "FieldIO.h"
+#include <chrono>
+#include <ctime>
+#include <algorithm>
+#include <iomanip>
+
+#include <fmt/compile.h>
+#include <fstream>
+#include "../domain/DomainData.h"
+
+
+FieldIO::FieldIO(const std::string &xml_filename) {
+#ifndef BENCHMARKING
+    m_logger = Utility::create_logger(typeid(this).name());
```

Comment:
this line can be a member initializer lists

Reply:
475b423